README.md

```diff
@@ -118,3 +113,210 @@ This stack is IPv6-capable. Caveats:
+    However, the desec-stack nameserver is available on localhost port 5321.
+    To check if desec-stack is working as expected, you can query the desec-stack nameserver locally for any information that you saved using your API.
+
+       EMAIL=john@example.com
+       PASSWORD=insecure
+       http POST https://desec.${DOMAIN}/api/v1/auth/users/ email:=\"${EMAIL}\" password:=\"${PASSWORD}\"
+       TOKEN=$(http POST https://desec.${DOMAIN}/api/v1/auth/token/login/ email:=\"${EMAIL}\" password:=\"${PASSWORD}\" | jq -r .auth_token)
+       http POST https://desec.${DOMAIN}/api/v1/domains/ Authorization:"Token ${TOKEN}" name:='"test.example"'
+       http POST https://desec.${DOMAIN}/api/v1/domains/test.example/rrsets/ Authorization:"Token ${TOKEN}" type:=\"A\" ttl:=60 records:='["127.0.0.254"]'
+
+    After registering a user with your API, creating a domain and publishing some info to the DNS, use
+
+       dig @localhost -p 5321 test.example 
+
+    to see if the nameserver is behaving as expected.
+
+1. **(Optional) Configure PyCharm for API Development.** As a docker-compose application, desec-stack takes a while to start.
+    Additionally, it is hard to connect a debugger to the docker containers.
+    Our recommended solution is to develop the API using Django tests running outside the docker-compose application.
+    This will dramatically decrease the time required for running the Django tests and enable just-in-time debugging in PyCharm.
+    Also, it will enable you to browse dependencies code within PyCharm and thus ease debugging.
+
+    1. To get started, we create a virtual python environment that (to some extend) mimics the python environment in the docker container.
+        In the project root,
+
+           cd api
+           python3 -m venv venv
+           source venv/bin/activate
+           pip install wheel
+           pip install -r requirements.txt
+
+    1. At this point, Django is ready to run in the virtual environment created above.
+        There are two things to consider when running Django outside the container.
+        First, the environment variables as defined in the `.env` file need to be made available in the shell.
+        This can be done with
+
+           set -a && source ../.env && set +a
+
+        Second, the API needs to be configured to use a local database instead of the dbapi host.
+        (dbapi, of course, is unavailable outside the docker-compose application.)
+        To use a different database, use the `settings_quick_test.py` instead of the default `settings.py`:
+
+           DJANGO_SETTINGS_MODULE=api.settings_quick_test
+
+        Finally, you can manage Django using the `manage.py` CLI.
+        As an example, to run the tests, use
+
+           python3 manage.py test
+
+        The tests should take about 10 seconds.
+
+    1. Open PyCharm in the project root directory and open the settings.
```

Comment:
I read the following but did not try it as my previous comment may be a blocker.

Reply:
pycharm needs to be configured to have the appropriate environment variables, just like the shell in the preceding example